app/Console/Commands/PhotosAddedNotification.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Mail\PhotosAdded;
+use App\Models\Configs;
+use App\Models\Photo;
+use App\Models\User;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Mail;
+use Illuminate\Support\Facades\Storage;
+
+class PhotosAddedNotification extends Command
+{
+	/**
+	 * The name and signature of the console command.
+	 *
+	 * @var string
+	 */
+	protected $signature = 'lychee:photos_added_notification';
+
+	/**
+	 * The console command description.
+	 *
+	 * @var string
+	 */
+	protected $description = 'Send email notifications for newly added photos';
+
+	/**
+	 * Create a new command instance.
+	 *
+	 * @return void
+	 */
+	public function __construct()
+	{
+		parent::__construct();
+	}
+
+	/**
+	 * Execute the console command.
+	 *
+	 * @return int
+	 */
+	public function handle()
+	{
+		$settings = Configs::get();
+
+		if ($settings['new_photos_notification']) {
```

Comment:
Same as above, this and the two lines above need to be changed for:
```suggestion
		if (Configs::get_Value('new_photos_notification','1') == '1') {
```
This ensure that you don't re-query the config and this provides a default value.

Reply:
I've gone with a default of 0 instead, hope that's ok.

Reply:
Also added an extra bit around the link that it puts together. Found it would 404 if there was two slashes '//' after the config url, so checked if there was a trailing slash first and added one if there wasn't. Not sure if that is the best way to go about that, so let me know if you can think of a better way.